scapy/layers/inet6.py

```diff
@@ -3995,3 +4024,4 @@ def _load_dict(d):
 bind_layers(IPv6, IPv6, nh=socket.IPPROTO_IPV6)
 bind_layers(IPv6, IP, nh=socket.IPPROTO_IPIP)
 bind_layers(IPv6, GRE, nh=socket.IPPROTO_GRE)
+bind_layers(IPv6, RPL, {"nh": 58, "dst": "ff02::1a"})
```

Comment:
You can use keywords, it's cleaner: `nh=58, dst=...`

Reply:
Cleaned up.